manager/testdata/new-blueprints/scenario4/plotter.yaml

```diff
@@ -192,16 +191,46 @@ spec:
       kind: M4DModule
       chart:
         name: ghcr.io/mesh-for-data/arrow-flight-config-module:0.1.0
+      capabilities:
+      - capability: read
+        scope: cluster
+        supportedInterfaces:
+        - source:
+          protocol: s3
+          dataformat: parquet
+        - source:
+          protocol: s3
+          dataformat: csv
     - name: transform-config
       kind: M4DModule
       chart:
         name: ghcr.io/mesh-for-data/m4d-arrow-flight-transform-conf:0.1.0
+      capabilities:
+      - capability: transform
+        scope: asset
```

Comment:
scope: workload

Reply:
I copied it from transform-config-module.yaml spec. it should be changed there too? IIUC the enablement of plugin is per service but applying them is per asset. @simanadler

Reply:
Applying plugins is beyond the scope. It is done internally by the service.

Reply:
@shlomitk From what I understood applying the plugin is done by the blueprint-controller.

Reply:
Blueprint controller deploys the plugins. If 10 different assets need to have a column redacted, the redact plugin will be deployed only once and called 10 times by the service. A config module will contain the specifics per an asset (action to perform, arguments,...)

Reply:
@shlomitk1 - regarding - A config module will contain the specifics per an asset (action to perform, arguments,...) - what should be the config module in this case? is it what we send to arrow-flight-config-module? Thanks

Reply:
Can we assume the following what scope means: 

Scope of  service/deployment module:

Asset - deployment per asset
Cluster - deployment per cluster
Workload- deployment per workload

Scope of config module depends on the scope of the service it configures :if the service is per asset than the config module will be per asset otherwise the config module will be per workload for each cluster.

plugin: per service.

Reply:
> Can we assume the following what scope means:
> 
> Scope of service/deployment module:
> 
> Asset - deployment per asset
> Cluster - deployment per cluster
> Workload- deployment per workload

Yes, agreed

> Scope of config module depends on the scope of the service it configures :if the service is per asset than the config module will be per asset otherwise the config module will be per workload for each cluster. (assuming we do not do cross workload deployment of config modules)

So you are saying that a config module will never be used across workloads.  I think it makes sense, certainly for now.  We may want to revisit it in the future.

> 
> plugin: per service.

Makes sense.  @roee88 @cdoron Please confirm